go/adbc/driver/databricks/stmt_record_reader.go

```diff
@@ -79,7 +80,6 @@ type statementReader struct {
 	StatementId string
 
 	// Fields from the execution response manifest:
-
```

Comment:
Need to keep the line to make it clear that the comment is about all field. If `go fmt` is dropping it, add a `//` where there is a blank.